utiliti/src/main/java/de/gurkenlabs/utiliti/components/Editor.java

```diff
@@ -820,7 +820,7 @@ private void saveMaps() {
       if (preferences().syncMaps()) {
         for (String file : FileUtilities.findFilesByExtension(
             new ArrayList<>(),
-            Paths.get(FileUtilities.combine(this.getProjectPath(), "maps")),
+            Paths.get(Resources.getLocation("maps").getPath()),
```

Comment:
@nightm4re94 ,

1. **Regarding the Path** - 
- So, the `Resources.getLocation("maps")` - returns `URL` type and the `FileUtilities.findFilesByExtension` function expects `Path` type in the second argument. So, `Resources.getLocation("maps").getPath()` method which returns String works as a bridge to convert `URL` -> `String` -> `Path` via the `Paths.get`. So, as the `Resources.getLocation("maps").getPath()` return `String`, the  `Paths.get` is needed. 

- Alternatively I could have done URL -> URI -> Path, but the `toURI()` method on URL throws exception when the path does not exists. whereas the the `getPath()` which returns the path part of the URL returns empty string on invalid URL.  I felt the `getPath()` seems to be better here as the `Resources.getLocation()` will never return an invalid URL as it is written fail-safely and there is no need of explicit redundant `try-catch` block needed for `toURI()` method otherwise. Let me know your thoughts. Thank you!

2. The `Resources.getLocation()` method first checks if it is available on the resource folder, if not it moves to the root folder of the project. So, it remains compatible with the old approach of having maps directory in the project root.
```
public static URL getLocation(String name) {
    URL fromClass = ClassLoader.getSystemResource(name);
    if (fromClass != null) {
      return fromClass;
    }
    try {
      return new URL(name);
    } catch (MalformedURLException e) {
      try {
        return (new File(name)).toURI().toURL();
      } catch (MalformedURLException e1) {
        return null;
      }
    }
  }
```

Thank you!

Reply:
haha, I guess this further highlights the importance of unifying path handling in the engine :D
You're completely right about my first remark, I was assuming `getPath` was actually returning a `Path` object rather than a String. However, since you're already changing this method, I suggest using `Path.of` instead of `Paths.get` which is behaving equivalently but was later introduced for API consistency.
Thanks for the clarifications with the `getLocation` approach - I think that method should also be adjusted in #817 to handle Paths instead of URLs.

Reply:
@nightm4re94  Done.